brave/brave-core
# Updating empty wallet message for unsupported regions

Fixes brave/brave-browser#6604

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
1. Enable rewards
2. Wait for initilization
3. Close browser
4. Open Preferences file and search for countryid_at_install
5. Replace current value with 19024
6. Restart Browser
7. Navigate to `brave://rewards`
8. Confirm the page wallet empty message conforms to the spec in the issue.

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
